### DISCUSSION TITLE
unable to Upload file in electron app

### DISCUSSION THREAD

**Discussion by Siva1212Copado** [webdriverio/webdriverio#13544] [Q&A]
Hi Team,
I am automating the electron app using Webdriver IO, I have a functionality where I have to upload the file and proceed, but when I was performing the this step with the below code it was failing with 
"ERROR webdriver: Request failed with status 400 due to element not interactable: element not interactable"

below is the reference which is using to upload a file:

const fileInput = await browser.$("file locator");
const filePath = 'Path to file'
const remoteFilePath = browser.uploadFile(filePath);
 fileInput.setValue(remoteFilePath);

There is no submit button to upload file.

**Comment by christian-bromann** [comment]
It is hard to guess what is going on without the ability to inspect the application under test. Make sure that the `file locator` actually matches an input element. Also note that the `uploadFile` command is a Selenium Grid feature and testing Electron apps usually only involves a Chromedriver. Generally I would avoid testing the upload process and make a API request to the backend server instead of using the file upload dialog since it is a native OS feature.
